sermant-agentcore/sermant-agentcore-config/config/config.properties

```diff
@@ -28,6 +28,8 @@ agent.service.inject.enable=true
 agent.service.dynamic.config.enable=true
 # HTTP server switch
 agent.service.httpserver.enable=false
+# xDS service discovery switch
+agent.service.xds.service.discover.enable=false
```

Comment:
Is this switch just for service discovery? I think it should be xDS service switch

Reply:
besides，use service discovery instead of service discover

Reply:
I think service discovery switch can implement more refined control.